### PR TITLE
Pretext map

### PR DESCRIPTION
**Pull request by EdHarry**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pretextmap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  sha256: d8667aba7013ea24548d71795c84dbe6482ac9ff7f77b88552112733ebaed700
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # [linux64]
+    - make
+    - clang>=9.0.0
```
Comment (dpryan79) [inline recipes/pretextmap/meta.yaml]: I assume earlier clang versions don't work, yes?

Comment (EdHarry) [reply recipes/pretextmap/meta.yaml]: That's right,
I've only tested myself on clang 9 and clang 10, but I know clang 4 is too old

Comment (EdHarry) [reply recipes/pretextmap/meta.yaml]: Specifically, clang 4 is missing the __get_cpuid_count intrinsic

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
Package(s) built on CircleCI are ready for inspection:

Arch | Package | Repodata
-----|---------|---------
linux-64 | [pretextmap-0.1.1-hc9558a2_0.tar.bz2](https://83372-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages/linux-64/pretextmap-0.1.1-hc9558a2_0.tar.bz2) | [repodata.json](https://83372-42372094-gh.circle-artifacts.com/0/tmp/artifacts/images/pretextmap%3A0.1.1--hc9558a2_0.tar.gz)
osx-64 | [pretextmap-0.1.1-0.tar.bz2](https://83378-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages/osx-64/pretextmap-0.1.1-0.tar.bz2) | [repodata.json](https://83378-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages/noarch/repodata.json)
***

You may also use `conda` to install these:

 * For packages on linux-64:
```
conda install -c https://83372-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages <package name>
```
 * For packages on osx-64:
```
conda install -c https://83378-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages <package name>
```
***

Docker image(s) built:

Package | Tag | Install with `docker`
--------|-----|----------------------
[pretextmap](https://83372-42372094-gh.circle-artifacts.com/0/tmp/artifacts/images/pretextmap%3A0.1.1--hc9558a2_0.tar.gz) | 0.1.1--hc9558a2_0 | <details><summary>show</summary>`curl "https://83372-42372094-gh.circle-artifacts.com/0/tmp/artifacts/images/pretextmap%3A0.1.1--hc9558a2_0.tar.gz" \| gzip -dc \| docker load`

**Pull request merged**